knowlg-app/proxy.config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "/api/*": {
```

Comment:
Why you defined the separate proxy route for each path ?
 Why can't we use existing proxy config file here ?

Reply:
Created proxy config js file